04 Abstraction Exercises Part 1/hotel-room-application/src/main/java/Main.java

```diff
@@ -0,0 +1,62 @@
+import com.hotel.room.application.HotelRoom;
+
+import java.util.Scanner;
+
+/**
+ * This is the Main Class
+ * */
+public class Main {
+    public static void main(String[] args){
+        int NumOfGuest, NumOfDays;
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Welcome to Hotel De Luna!");
+
+        int option;
+        do {
+            System.out.println("Choose an Option: ");
+            System.out.println("1. Book Reservation: ");
+            System.out.println("2. Cancel Reservation: ");
+            System.out.println("0. Exit");
+            System.out.print("Enter your Choice:");
+            option = scanner.nextInt();
+
+            switch (option) {
+                case 1: {
+                    System.out.println("Enter Number of Guest: ");
```

Comment:
What if the room is reserved already?

Reply:
pag may naka check in po eh bawal na po ng ibang guest na mag check in

Reply:
para po malaman eh kailangan po na may mag check if may na una na nagcheck in hehe. wala pa po akong validation sir gagawin palang po